src/domain.module.ts

```diff
@@ -52,13 +52,19 @@ import { IHumanDescriptionRepository } from '@/domain/human-description/human-de
 import { HumanDescriptionRepository } from '@/domain/human-description/human-description.repository';
 import { BalancesValidator } from '@/domain/balances/balances.validator';
 import { BalancesApiModule } from '@/datasources/balances-api/balances-api.module';
+import { ILockingRepository } from '@/domain/locking/locking.repository.interface';
+import { LockingRepository } from '@/domain/locking/locking.repository';
+import { RankValidator } from '@/domain/locking/rank.validator';
+import { LockingEventValidator } from '@/domain/locking/locking-event.validator';
+import { LockingApiModule } from '@/datasources/locking-api/locking-api.module';
 
 @Global()
 @Module({
   imports: [
     BalancesApiModule,
     ConfigApiModule,
     HumanDescriptionApiModule,
+    LockingApiModule,
```

Comment:
I would leave this module out of the `DomainModule` for now: the most recent approach to have smaller modules is preferable from a maintenance perspective (see `AccountDataSourceModule`).

This is also to say that we should break down this module into smaller ones in the future.

Reply:
Removed in 53082b79be2c075827de2f2c059f984030fbe1b8.

Reply:
A domain-specific module was added in the follow-up PR in 43c1d35a03d799bffdeb4b50eaf2d0342e9b006c, connecting everything.